src/js/count.js

```diff
@@ -0,0 +1,80 @@
+class Count {
+	/**
+	 * Class constructor.
+	 *
+	 * @param {HTMLElement} [countEl] - The component element in the DOM
+	 * @param {Object} [opts={}] - An options object for configuring the component
+	 */
+	constructor (countEl, opts) {
+		this.countEl = countEl;
+		this.options = Object.assign({}, {}, opts || Count.getDataAttributes(countEl));
+		this.articleId = this.options.articleId;
+	}
+
+	/**
+	 * Render a comment count instance.
+	 *
+	 * @access private
+	 * @returns {HTMLElement}
+	 */
+	renderCount () {
+		if (this.countEl && !(this.countEl instanceof HTMLElement)) {
+			this.countEl = document.querySelector(this.countEl);
+		}
+
+		if (!this.countEl) {
+			throw new Error('Element must be a HTMLElement');
+		}
+
+		return Count.fetchCount(this.articleId)
+			.then((count) => {
+				this.countEl.innerHTML = count;
```

Comment:
If this is only placing text into the count element, it would be better to use .textContent as it is specifically for text and not HTML content. It has the benefit of not need ing to be parsed as HTML (more performant) and avoids any potential cross-site-scripting vulnerabilities (more secure).

```suggestion
				this.countEl.textContent = count;
```
Documentation for textContent: https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent

Reply:
Thanks for explaining that to me 😄 I didn't know that - yes, in this case we're only placing text in the element so I'll switch to use `textContent`.